bookwyrm/templates/403.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'layout.html' %}
+{% load i18n %}
+{% load utilities %}
+
+{% block title %}{% trans "Oh no!" %}{% endblock %}
+
+{% block content %}
+<div class="block">
+    <h1 class="title">{% trans "Permission Denied" %}</h1>
+    {% blocktrans trimmed with level=request.user|get_user_permission %}
+    <p class="content">You do not have permission to view this page or perform this action. Your user permission level is <code>{{ level }}</code>.</p>
```

Comment:
It'll work better to wrap just the text in two separate blocktrans tags, rather than including both sentences with the markup, since it will be hard to translate with the html in place

Reply:
Like this?

```django
    {% blocktrans trimmed with level=request.user|get_user_permission %}
    <p class="content">You do not have permission to view this page or perform this action. Your user permission level is <code>{{ level }}</code>.</p>
    {% endblocktrans %}
    <p class="content">{% trans "If you think you should have access, please speak to your BookWyrm server administrator." %}</p>

```

Reply:
Almost; the less markup included the better:
```
    <p class="content">
    {% blocktrans trimmed with level=request.user|get_user_permission %}
    You do not have permission to view this page or perform this action. Your user permission level is <code>{{ level }}</code>.
    {% endblocktrans %}
    </p>

    <p class="content">{% trans "If you think you should have access, please speak to your BookWyrm server administrator." %}</p>
```

Reply:
ahh, got it.